docs(progress-bar-expressive): fix noisy status updates on docs page

cordeliadillon opened the pull request:
<!-- Insert GitHub issue number below -->
Fixes #2311 

<!-- Select which type of PR this is -->
- [ ] This PR contains CSS changes
- [x] This PR does not contain CSS changes

## Description
The message elements with `role="status"` in the expressive progress bar doc examples are reading out constantly when perusing the Skin docs with a screen reader. This update sets `aria-live="off"` on the rendered examples (but not on the rendered code snippets) so as not to be so disruptive when someone is reading the Skin docs.

## Notes
I opted to use `aria-live="off"` rather than removing `role="status"`, since we use the status role as a selector in the `main.js` file. Setting an explicit `aria-live="off"` overrides status's implicit `aria-live="polite"`.

## Screenshots
N/A

## Checklist
<!-- Acknowledge completion of steps in checklists below. Delete lists that are not applicable -->

<!-- For all PR types -->
- [x] I verify the build is in a non-broken state
- [x] I verify all changes are within scope of the linked issue

saiponnada reviewed (approved):
Looks good to me. One minor suggestion. It's better to add a note to docs on why we added `aria-live="off"` so that it might be clear to someone reading these understand why role=status is still not announcing the messages

agliga reviewed (changes requested):
LGTM overall

However I see some docs files were changed. Usually those should not be changed on docs updates. You might need to clean your node_modules and update from 17.4.0

cordeliadillon commented:
> Looks good to me. One minor suggestion. It's better to add a note to docs on why we added `aria-live="off"` so that it might be clear to someone reading these understand, why `role=status` is not announcing the messages

Great suggestion, @saiponnada! I updated the docs and re-requested your review. :)

saiponnada reviewed (approved):
Perfect!

cordeliadillon commented:
@saiponnada @agliga, thanks for the re-review! Feel free to merge when ready. (I don't have merge access.)
